src/app/pages/OnDemandTvPage/index.jsx

```diff
@@ -173,7 +173,7 @@ const OnDemandTvPage = ({ pageData }) => {
           margins={getGroups(true, true, true, true, false, false)}
         >
           <VisuallyHiddenText as="h1" tabIndex="-1" id="content">
-            {brandTitle}, {formattedTimestamp}
+            {`${brandTitle}, ${formattedTimestamp}`}
```

Comment:
I just had a look at the on demand radio page - how come we're not using the `OnDemandHeadingBlock`, as per https://github.com/bbc/simorgh/blob/cee4242130bd5fcc8e717d4e9845b346dbc82010/src/app/pages/OnDemandRadioPage/index.jsx#L166

Reply:
https://github.com/bbc/simorgh/pull/7127/files#diff-47a8bccb05abace253220afda8c197bdR193

Reply:
Hmm GitHubt can't handle links when the file isn't expanded. See line 193

Reply:
Ah yes, I see it now. How come this is not an issue on the On Demand Radio page then? (Can't see the same functionality)

Reply:
The requirements for the two are quite different.
ODTV:
- visually hidden heading needs to be on top of page
- visible heading should not be an h1

OD Radio
- only one heading required

Reply:
Thanks for explaining 👍